### PR TITLE
Update ISA JSON compliance documentation

### PR DESCRIPTION
**Pull request by somathias**
I was not able to reopen the previous PR [#1812](https://github.com/seek4science/seek/pull/1812) so I have turned my suggested changes into a new PR.

This PR includes
- fixed typos
- added info boxes about features needed to be enabled regarding the ISA-JSON compliant experiments feature as well as a general disclaimer that FAIRDOM-SEEK instances are customisable.
- regrouped the information about Experiment Sample Templates and Samples within ISA-JSON compliant experiments

### CODE REVIEW

File: help/user-guide/browsing.md [hunk 1]
```diff
@@ -26,10 +26,15 @@ The columns to be shown by the table view can be customised, adding/removing col
 ![Table view configuration](/images/user-guide/browsing_table_config.png){:.screenshot}
 
 # Browsing Samples
-You can browse Samples registered in SEEK by Sample Type or by Templates.
+You can browse Samples registered in SEEK by Sample Type or by Experiment Sample Templates.
 * From any page, click on Browse from the top navigation bar
 * Select Samples from the dropdown menu
```
Comment (NielsGeudens) [inline help/user-guide/browsing.md:32]: Suggestion to add the following bullet points: 
* To query by Experiment Sample Templates, click the button on the top right of the screen
* To filter by Sample Type, use the filters on the left.

File: help/user-guide/create-sample-isajson-compliant.md [hunk 2]
```diff
@@ -3,59 +3,67 @@ title: SEEK User Guide - Create Sample in ISA-JSON compliant Experiments
...
-Sources table and Samples table are interactive tables (dynamic tables) that allow samples creation and editing. 
-* Studies contain both Sources table and Samples table.
-* Assays contains only Samples table.
+## Types of samples in ISA-JSON compliant Experiments
 
-## samples
-In Experiment View, you can also view Study Sources, Study Samples, and Assay Samples in a searchable table by selecting "samples (n)" from the tree view on the left sidebar. Samples cannot be created or edited via this view.
+Study Source(s)
+* Study Sources must be created within an ISA Study, using an Experiment Sample Template level "Study Source".
+* Each Study Source must be the input of at least one Study Sample (or more) in the same ISA Study.
```
[the remark this answers is not in the document]
Comment (floradanna) [reply help/user-guide/create-sample-isajson-compliant.md:12]: you are right. better rephrase it as
* Each Study Source should be the input of at least one Study Sample (or more) in the same ISA Study.

File: help/user-guide/create-sample-isajson-compliant.md [hunk 3]
```diff
@@ -3,59 +3,67 @@ title: SEEK User Guide - Create Sample in ISA-JSON compliant Experiments
...
-Samples in ISA-JSON compliant Experiments can be visualised in the "design" tab (Study or Assay design). Samples are grouped in tables: Study Sources table, Study Samples table, Assay Samples table, Experiment overview.
+# Working with Samples in ISA-JSON compliant Experiments
 
-## Sources table and Samples table
-Sources table and Samples table are interactive tables (dynamic tables) that allow samples creation and editing. 
-* Studies contain both Sources table and Samples table.
-* Assays contains only Samples table.
+## Types of samples in ISA-JSON compliant Experiments
 
-## samples
```
Comment (floradanna) [inline help/user-guide/create-sample-isajson-compliant.md:14]: @somathias why do you suggest to delete these paragraphs?

File: help/user-guide/create-sample-isajson-compliant.md [hunk 4]
```diff
@@ -3,59 +3,67 @@ title: SEEK User Guide - Create Sample in ISA-JSON compliant Experiments
 layout: page
 ---
 
-# Viewing samples in ISA-JSON compliant Experiments
-Samples in ISA-JSON compliant Experiments can be visualised in the "design" tab (Study or Assay design). Samples are grouped in tables: Study Sources table, Study Samples table, Assay Samples table, Experiment overview.
+# Working with Samples in ISA-JSON compliant Experiments
```
Comment (floradanna) [inline help/user-guide/create-sample-isajson-compliant.md:6]: In this page, viewing samples and creating samples are two different actions. Why do you suggest to do not keep the distinction?

File: help/user-guide/create-sample-isajson-compliant.md [hunk 5]
```diff
@@ -3,59 +3,67 @@ title: SEEK User Guide - Create Sample in ISA-JSON compliant Experiments
...
-* Assays contains only Samples table.
+## Types of samples in ISA-JSON compliant Experiments
 
-## samples
-In Experiment View, you can also view Study Sources, Study Samples, and Assay Samples in a searchable table by selecting "samples (n)" from the tree view on the left sidebar. Samples cannot be created or edited via this view.
+Study Source(s)
+* Study Sources must be created within an ISA Study, using an Experiment Sample Template level "Study Source".
+* Each Study Source must be the input of at least one Study Sample (or more) in the same ISA Study.
 
-![dynamic table isa study source](/images/user-guide/isajson-compliance/dynamictable_isastudy_source.png){:.screenshot}
```
Comment (floradanna) [inline help/user-guide/create-sample-isajson-compliant.md:17]: is the image moved?

File: help/user-guide/isa-json-compliant-experiment.md [hunk 6]
```diff
@@ -49,69 +54,33 @@ The outputs of an Assay can only be used as inputs by the next Assay in the same
 * data files (e.g. files containing measurements, rawdata.fastq, processed data, reads.counts.txt).
 
 
-# Experiment Sample Templates
-
-Experiment Sample Templates act as blueprints to create Sample Types within ISA Studies and ISA Assays. The same Experiment Sample Template can be applied multiple times to create Sample Types in different ISA Studies or ISA Assays.
+## Experiment Sample Templates
```
Comment (floradanna) [inline help/user-guide/isa-json-compliant-experiment.md:57]: @somathias why shouldn't this be main title (only 1 #)?

File: help/user-guide/isa-json-compliant-experiment.md [hunk 7]
```diff
@@ -49,69 +54,33 @@ The outputs of an Assay can only be used as inputs by the next Assay in the same
...
-    * Repository or data archive (e.g. EBI databases)
-    * SEEK Project (Project specific templates)
+See [Experiment Sample Templates](isajson-templates.html) for more information.
 
-3. Organism: optional free text
 
-## ISA minimal starter template
+## Samples in ISA-JSON compliant Experiments
 
-When the ISA-JSON compliance feature is enabled in the platform, it is possible to use the "ISA minimal starter template" as a starting point to create any Experiment Sample Template. One ISA minimal starter template for each ISA Level is provided with the feature.
```
Comment (floradanna) [inline help/user-guide/isa-json-compliant-experiment.md:79]: why do you think this paragraph should be removed?

File: help/user-guide/isa-json-compliant-experiment.md [hunk 8]
```diff
@@ -49,69 +54,33 @@ The outputs of an Assay can only be used as inputs by the next Assay in the same
...
-1. One ISA Level
-    * Study Source
-    * Study Sample
-    * Assay material
-    * Assay data file
-
-2. One Repository name or SEEK Project
-    * Repository or data archive (e.g. EBI databases)
-    * SEEK Project (Project specific templates)
+See [Experiment Sample Templates](isajson-templates.html) for more information.
```
Comment (floradanna) [inline help/user-guide/isa-json-compliant-experiment.md:64]: ok to delete the characteristics and link to isajson-templates page

File: help/user-guide/isa-json-compliant-experiment.md [hunk 9]
```diff
@@ -49,69 +54,33 @@ The outputs of an Assay can only be used as inputs by the next Assay in the same
...
-* Study Samples must be the outputs of a sampling protocol applied to ISA Study Sources, in the same Study.
-* Each Study Sample must be the output of at least one Source (or more), in the same ISA Study.
-
-Material output assay sample(s)
-* Assay material samples must be created within an ISA Assay, using an Experiment Sample Template level "Assay - material".
-* Assay material samples must be the outputs of a protocol applied to the inputs of the Assay.
-* Each Assay Sample must have at least one input (or more). Inputs can be: 
-  * Study Samples in the same ISA Study;
-  * Assay samples from one preceding Assay, in the same Assay Stream.
+See [Working with Samples in ISA-JSON compliant Experiments](create-sample-isajson-compliant.html) for more information.
```
Comment (floradanna) [inline help/user-guide/isa-json-compliant-experiment.md:77]: The linked page gives different info. I would not link it here.

File: help/user-guide/isa-json-compliant-experiment.md [hunk 10]
```diff
@@ -49,69 +54,33 @@ The outputs of an Assay can only be used as inputs by the next Assay in the same
...
-In ISA-JSON compliant Experiments, samples must be created within a Sample Type derived from an Experiment Sample Template, associated with one ISA Studies or one ISA Assays.
+In ISA-JSON compliant experiments, samples must be created within a Sample Type derived from an Experiment Sample Template, associated with one ISA Study or one ISA Assay.
 
 The "level" of the Experiment Sample Template applied to generate a Sample Type within an ISA Study or ISA Assay determines the type of samples that will be created. Specifically, 
 * ISA Study Source Template for Study Sources
 * ISA Study Sample Template for Study Samples
 * ISA material output Assay Template for material samples
 * ISA data file Assay Template for digital data file samples
 
-## Types of samples in ISA-JSON compliant Experiments
```
Comment (floradanna) [inline help/user-guide/isa-json-compliant-experiment.md:91]: please, keep this paragraph. Different types of samples are explained here and nowhere else

File: help/user-guide/isajson-templates.md [hunk 11]
```diff
@@ -2,28 +2,38 @@
 title: SEEK User Guide - Experiment Sample Templates
 layout: page
 ---
+# Experiment Sample Templates
```
Comment (floradanna) [inline help/user-guide/isajson-templates.md:5]: I agree with moving here the text related to Templates that was in isa-json-compliant-experiment

File: help/user-guide/isajson-templates.md [hunk 12]
```diff
@@ -2,28 +2,38 @@
...
+   * Assay material
+   * Assay data file
+
+2. One Repository name or SEEK Project
+   * Repository or data archive (e.g. EBI databases)
+   * SEEK Project (Project specific templates)
+
+3. Organism: optional free text
 
-## ISA minimal starter template
```
Comment (floradanna) [inline help/user-guide/isajson-templates.md:15]: please, keep it

File: help/user-guide/isajson-templates.md [hunk 13]
```diff
@@ -2,28 +2,38 @@
...
-When the ISA-JSON compliance feature is enabled in the platform, it is recommended to use the "ISA minimal starter template" as the starting point for creating any new Experiment Sample Template. One ISA minimal starter template is provided for each ISA Level with the feature.
 
-<!--where to find them, specification-->
+## Platform-wide and project specific Experiment Sample Templates
 
-# For project members
```
Comment (floradanna) [inline help/user-guide/isajson-templates.md:20]: please, keep the distinction "for instance administrator" and "for project members" as main title (1 #).

File: help/user-guide/isajson-templates.md [hunk 14]
```diff
@@ -36,4 +46,19 @@ Project members have the capability to create Experiment Sample Templates for th
 * If you want to add new attributes of your choice to your new template, select “Add new attribute” button.
 * Fill out the mandatory and optional fields. Note that for ISA-JSON compliant Experiments, the ISA Tag is a mandatory field.
 * For ISA tag, note that the following tags can only be present once in a template: source, sample, other_material, data_file, protocol. One of these tags is already present in existing templates, so select one of the following as ISA tags: source_characteristic, parameter_value, sample_characteristic, other_material_characteristic, data_file_comment.
-6. Select "Create".
\ No newline at end of file
+6. Select "Create".
+
+
+### JSON upload via Server admin panel [FOR SYSTEM ADMINISTRATORS]
```
Comment (floradanna) [inline help/user-guide/isajson-templates.md:52]: please, keep the distinction "for instance administrator" and "for project members" as main title (1 #).

File: help/user-guide/isa-json-compliant-experiment.md (hunk 6 above)
Comment (somathias) [reply help/user-guide/isa-json-compliant-experiment.md:57]: Personally, I prefer if "ISA-JSON compliant investigation" is the main title and all other information in the same file is organised under it at suitable levels. (Formulated in a different way: why did you have multiple main titles in the same .md file?)

File: help/user-guide/isajson-templates.md (hunk 12 above)
Comment (somathias) [reply help/user-guide/isajson-templates.md:15]: It is only moved to the end of the file, not deleted.

File: help/user-guide/create-sample-isajson-compliant.md (hunk 4 above)
Comment (somathias) [reply help/user-guide/create-sample-isajson-compliant.md:6]: To me both fall under "Working with samples". The distinction is hence still there, I just felt an umbrella heading for the page was beneficial for the flow of information.

File: help/user-guide/create-sample-isajson-compliant.md (hunk 3 above)
Comment (somathias) [reply help/user-guide/create-sample-isajson-compliant.md:14]: Nothing is deleted, only moved.

File: help/user-guide/create-sample-isajson-compliant.md (hunk 4 above)
Comment (floradanna) [reply help/user-guide/create-sample-isajson-compliant.md:6]: ok

File: help/user-guide/create-sample-isajson-compliant.md (hunk 3 above)
Comment (floradanna) [reply help/user-guide/create-sample-isajson-compliant.md:14]: ok

**Review by floradanna — APPROVED** [review]

**Comment by floradanna** [comment]
@stuzart 
@NielsGeudens and I reviewed the content with @somathias and this PR is now ready to be merged. I will then check again for broken links, in case. Can you merge it or should I do it?

**Comment by stuzart** [comment]
> @stuzart @NielsGeudens and I reviewed the content with @somathias and this PR is now ready to be merged. I will then check again for broken links, in case. Can you merge it or should I do it?

Please feel free to merge, for there is a conflict that needs resolving. Let me know if you have trouble resolving it through the github UI

**Comment by somathias** [comment]
@floradanna I took the liberty of resolving the conflict by adding the additional sentence referencing where to find the EBI specific templates to the section for system administrators in the isajson-templates.md.

**Comment by floradanna** [comment]
Thanks a million @somathias ! I am going to merge it now.
@kdp-cloud can we pull and merge these changes into our sandbox and production before the training (maybe also test)?

**Pull request merged**